src/BuildSamples/SourceLink/Directory.Build.targets

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Copy the output assembly and its PDB to the appropriate Pass/Fail folder. -->
+  <PropertyGroup>
+    <!-- EnableSourceLink is set in Microsoft.SourceLink.Common.props -->
```

Comment:
>props [](http://example.com/codeflow?start=64&length=5)

Just curious, and this target file will naturally be on the path for certain modern VS version users?

Reply:
Yes. It's been so long that I don't remember when support for Directory.Build.props and Directory.Build.targets was added to MSBuild. Definitely there since VS 2017. Likely longer.